spring-multirabbit/src/test/java/org/springframework/boot/autoconfigure/amqp/MultiRabbitConnectionFactoryCreatorTest.java

```diff
@@ -100,11 +103,12 @@ void shouldInstantiateRoutingConnectionFactory() throws Exception {
         externalWrapper.addConnectionFactory(DUMMY_KEY, connectionFactory1, containerFactory, rabbitAdmin);
 
         when(springFactoryCreator.rabbitConnectionFactory(rabbitProperties, resourceLoader, credentialsProvider,
-                credentialsRefreshService, connectionNameStrategy)).thenReturn(new CachingConnectionFactory());
+                credentialsRefreshService, connectionNameStrategy,
+                connectionFactoryCustomizers)).thenReturn(new CachingConnectionFactory());
```

Comment:
you can use the previous line as much as possible, and leave only .thenReturn(...); at the new line.